src/components/UserDetailSections/UserAccounts/UserAccounts.js

```diff
@@ -51,14 +63,28 @@ const UserAccounts = ({
   useEffect(() => {
     const open = records.filter(account => account?.status?.name !== accountStatuses.CLOSED);
     const closed = records.filter(account => account?.status?.name === accountStatuses.CLOSED);
-    const openTotal = open.reduce((acc, { remaining }) => (acc + parseFloat(remaining)), 0);
+    const refunded = records.filter(account => (account?.paymentStatus?.name === refundStatuses.RefundedFully || account?.paymentStatus?.name === refundStatuses.RefundedPartially));
+    const loansClaim = get(resources, ['loansHistory', 'records'], []).filter(loan => loan?.action === loanActions.CLAIMED_RETURNED);
```

Comment:
As in the line above with `account?`, you can avoid `_.get(...)` with code like `resources.loansHistory.records?.filter(...)`. There is no need to refactor existing code, but in general we are moving toward this [optional chaining syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).

Reply:
Ok Zack I will remember and I preferred to update the code.